react/react-training/src/screens/Game.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+
+import Board from './MyScreenComponent/Board';
+import './Game.css';
+
+function calculateWinner(squares) {
+  const lines = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]];
+  for (let i = 0; i < lines.length; i += 1) {
+    const [a, b, c] = lines[i];
+    if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
+      return squares[a];
+    }
+  }
+  return null;
+}
+
+class Game extends Component {
+  constructor(props) {
```

Comment:
You do not need this constructor, you can replace it by moving the definition of the state outside

Reply:
I don't understand what do you mean here.

Reply:
Remember the talk